docs/getting-started.rst

```diff
@@ -0,0 +1,39 @@
+.. _getting-started:
+
+Getting started
+===============
+
+If you don't have Portia running yet, please read the :ref:`Installation guide <installation>` first.
+
+Creating a project and spider
+-----------------------------
+
+To create a new project, begin by entering the site's URL in the navigation bar at the top of the page and clicking ``Start``. Portia can be used as a web browser, and you can navigate to pages you want to extract data from and create templates for them. Clicking ``Start`` should create a new project along with a spider for the website, and you should see the loaded web page:
+
+.. image:: _static/portia-new-project.png
+    :alt: Newly created project
+
+The spider can be configured on the right. The start pages are the URLs the spider will visit when beginning a new crawl. Portia can be used as a web browser, and you can navigate to the pages you want to extract data from and create new templates. To define the data you wish to extract from the page, click the ``Annotate this page`` button, which will create a new template and allow you to annotate the page.
```

Comment:
`Portia can be used as a web browser, and you can navigate to pages you want to extract data from` is repeated from above

Use the word `sample` instead of template